### PR TITLE
Update MSTest.Sdk to handle playwright

### PR DESCRIPTION
**Pull request by Evangelink**
When using Playwright, the user is expecting a different set of NuGet packages to be set by default.

### CODE REVIEW

File: src/Package/MSTest.Sdk/Sdk/Runner/Common.targets [hunk 1]
```diff
@@ -26,4 +26,9 @@
     <MicrosoftTestingExtensionsCodeCoverageVersion Condition=" '$(MicrosoftTestingExtensionsCodeCoverageVersion)' == '' " >$(MicrosoftTestingExtensionsCodeCoverageVersion)</MicrosoftTestingExtensionsCodeCoverageVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePlaywright Condition=" '$(EnablePlaywright)' == '' ">false</EnablePlaywright>
+    <MicrosoftPlaywrightVersion Condition=" '$(MicrosoftPlaywrightVersion)' == '' ">$(MicrosoftPlaywrightVersion)</MicrosoftPlaywrightVersion>
```
Comment (Evangelink) [inline src/Package/MSTest.Sdk/Sdk/Runner/Common.targets]: Where is `MicrosoftPlaywrightVersion` default version set?

Comment (MarcoRossignoli) [reply src/Package/MSTest.Sdk/Sdk/Runner/Common.targets]: https://github.com/microsoft/testfx/blob/main/src/Package/MSTest.Sdk/Sdk/Sdk.props.template

Comment (Evangelink) [reply src/Package/MSTest.Sdk/Sdk/Runner/Common.targets]: Thanks, fixed.

File: src/Package/MSTest.Sdk/Sdk/Runner/Common.targets [hunk 2]
```diff
@@ -26,4 +26,9 @@
     <MicrosoftTestingExtensionsCodeCoverageVersion Condition=" '$(MicrosoftTestingExtensionsCodeCoverageVersion)' == '' " >$(MicrosoftTestingExtensionsCodeCoverageVersion)</MicrosoftTestingExtensionsCodeCoverageVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePlaywright Condition=" '$(EnablePlaywright)' == '' ">false</EnablePlaywright>
```
Comment (Evangelink) [inline src/Package/MSTest.Sdk/Sdk/Runner/Common.targets]: Shall this be in the sdk.props.template file?

Comment (MarcoRossignoli) [reply src/Package/MSTest.Sdk/Sdk/Runner/Common.targets]: I would put both in the prop as defaults yep

File: src/Package/MSTest.Sdk/Sdk/Sdk.props.template [hunk 3]
```diff
@@ -9,6 +9,7 @@
     <MSTestVersion Condition=" '$(MSTestVersion)' == '' ">${MSTestVersion}</MSTestVersion>
     <MSTestEngineVersion Condition=" '$(MSTestEngineVersion)' == '' ">${MSTestEngineVersion}</MSTestEngineVersion>
     <MicrosoftNETTestSdkVersion Condition=" '$(MicrosoftNETTestSdkVersion)' == '' ">${MicrosoftNETTestSdkVersion}</MicrosoftNETTestSdkVersion>
+    <MicrosoftPlaywrightVersion Condition=" '$(MicrosoftPlaywrightVersion)' == '' ">${MicrosoftPlaywrightVersion}</MicrosoftPlaywrightVersion>
```
Comment (MarcoRossignoli) [inline src/Package/MSTest.Sdk/Sdk/Sdk.props.template:13]: @Evangelink we need to add this env var to the templating inside the csproj of the package project

**Review by MarcoRossignoli — COMMENTED** [review]
Is it possible do some kind of tests of it?

**Pull request merged**